### DISCUSSION TITLE
Including LiveCharts2 into a UWP project

### DISCUSSION THREAD

**Discussion by iAkrobat** [Live-Charts/LiveCharts2#961] [Q&A]
Hi guys,

I am trying to add LiveCharts2 to a UWP / VB project for quiet some time now. Unfortunately I just can't seem to find a "simple" step-by-step explanation for this, and it feels like the more I try to understand how all this works / is supposed to work, the more questions I have.

On lvcharts.com under "Get started" I selected UWP
-but that does reference to UNO?
Fine, I go to the UNO documentary
-but that does not exist?
So I tried to go back one iteration, where I finally found a setup guide for an UNO Project
-but I don't want to create an UNO project?
Below the NuGet picture there is a link that sends me to the UNO setup guide.
So I followed the guide, imported the LiveChartsCore.SkiaSharpView.Uno NuGet package to the project.
I added  xmlns:lvc="using:LiveChartsCore.SkiaSharpView.Uno" to the xaml file in which I want to add the CartesianChart, and tried to create this chart, copying the guide exactly:

<Page.DataContext>
    <local:ViewModel />
</Page.DataContext>

<lvc:CartesianChart
    Series="{Binding Series}">
</lvc:CartesianChart>

But Visual Studio does not like the lvc:CartesianChart at all.
If I don't put the Chart into a Grid: "The specified value cannot be assigned. The following type was expected: "UIElement".
If i put the Chart into a Grid: "A value of type 'CartesianChart' cannot be added to a collection or dictionary of type 'UIElementCollection'"

So my questions are:
1. Why do I have to create an UNO project first?
2. Why do I need to start the project for a mobile app, when I just want to add a graph to a UPW project?
3. Do I have to use a C# Project behind the XAML?
4. Am I missing any NuGet Packages?
5. Help! ?

_Edit: formatting_

**Comment by Strypper** [comment]
have resolved this yet?
